src/models.ts

```diff
@@ -80,6 +81,13 @@ export const validateReportLoad = validate(loadSchema, {
   }
 });
 
+export interface ICreateReportConfiguration {
```

Comment:
all seems good.
please use IReportCreateConfiguration
to be like IReportLoadConfiguration